R/adults.R

```diff
@@ -6,22 +6,22 @@
 #' @param cross_channel_gates_closed Variable describing number of days gates are closed for each month
 #' @param prop_bay_trans Variable describing proportion transport to the bay
 #' @param prop_delta_trans Variable describing proportion transport to the delta
-#' @param .intercept Intercept, source: Empirical model fit using 2008–2011 tagging data provided by East Bay Municipal Utility District
-#' @param .wild Coefficient for \code{wild} variable, source: Estimated with coded wire tag data 2010–2013 \href{https://nrm.dfg.ca.gov/FileHandler.ashx?DocumentID=162355&usg= AOvVaw0VgMOwD7knFfSxRZy6k8RG}{(Kormos et al. 2012, Palmer-Zwahlen & Kormos 2013-2015, Palmer-Zwahlen et al. 2018)}
-#' @param .natal_flow Coefficient for \code{natal_flow} variable, source: Empirical model fit using 2008–2011 tagging data provided by East Bay Municipal Utility District
-#' @param .cross_channel_gates_closed Coefficient for \code{cross_channel_gates_closed} variable, Source: Empirical model fit using  2008–2011 tagging data provided by East Bay Municipal Utility District.
-#' @param .prop_bay_trans Coefficient for \code{prop_bay_trans} variable, source: Estimated with coded wire tag data 2010–2013 \href{https://nrm.dfg.ca.gov/FileHandler.ashx?DocumentID=162355&usg= AOvVaw0VgMOwD7knFfSxRZy6k8RG}{(Kormos et al. 2012, Palmer-Zwahlen & Kormos 2013-2015, Palmer-Zwahlen et al. 2018)}
-#' @param .prop_delta_trans Coefficient for \code{prop_delta_trans} variable, source: Estimated with coded wire tag data 2010–2013 \href{https://nrm.dfg.ca.gov/FileHandler.ashx?DocumentID=162355&usg= AOvVaw0VgMOwD7knFfSxRZy6k8RG}{(Kormos et al. 2012, Palmer-Zwahlen & Kormos 2013-2015, Palmer-Zwahlen et al. 2018)}
+#' @param .intercept Intercept
+#' @param .wild Coefficient for \code{wild} variable
+#' @param .natal_flow Coefficient for \code{natal_flow} variable
+#' @param .cross_channel_gates_closed Coefficient for \code{cross_channel_gates_closed} variable
+#' @param .prop_bay_trans Coefficient for \code{prop_bay_trans} variable
+#' @param .prop_delta_trans Coefficient for \code{prop_delta_trans} variable
```

Comment:
i'm thinking it would be nice to link to the `params` docs in each sub-model
```r
#' @details See \code{\link{params}} for more details on sources for coefficient values
```
i don't know that I got the syntax right on this

Reply:
added